<launch>
 <arg name="x_0" default="0"/>
 <arg name="y_0" default="0"/>
 <arg name="x_1" default="0"/>
 <arg name="y_1" default="0"/>

 <node pkg="map_server" type="map_server" name="map_server" args="$(find explore_stage_com)/bosch_maps/maze3.png 0.1" respawn="false" > 
    <param name="frame_id" value="/map" />
 </node>

	
 <group ns="robot_0">

    <node pkg="map_transform" type="visibility" name="visibility" output="screen" args="0">    
       <param name="infl" value="10" />
       <param name="defl" value="10" />
       <param name="debug" value="false" />
       <param name="ground_truth" value="false" />
       <param name="tf_prefix" value ="/robot_0" /> 
       <remap from="/robot_0/visibility/map" to="/map"/>
       <remap from="/robot_0/visibility/l_map" to="/robot_0/l_map"/>
    </node>

    <node pkg="tf" type="static_transform_publisher" name="fake_localize_0" args="$(arg x_0) $(arg y_0) 0 0 0 0 /map /robot_0/base_link 10"/>

 </group>

 <group ns="robot_1">

    <node pkg="map_transform" type="visibility" name="visibility" output="screen" args="0">    
       <param name="infl" value="6" />
       <param name="infl" value="6" />
       <param name="debug" value="false" />
       <param name="ground_truth" value="false" />
       <param name="tf_prefix" value ="/robot_1" /> 
       <remap from="/robot_1/visibility/map" to="/map"/>
       <remap from="/robot_1/visibility/l_map" to="/robot_1/l_map"/>
    </node>

    <node pkg="tf" type="static_transform_publisher" name="fake_localize_1" args="$(arg x_1) $(arg y_1) 0 0 0 0 /map /robot_1/base_link 10"/>

 </group>

 <node name="planner" pkg="map_transform" type="planner" output="screen"/>

 <node name="rviz" pkg="rviz" type="rviz" args="-d $(find map_transform)/launch/planning.rviz" />

</launch>
